glue_lab/handlers.py

```diff
@@ -13,7 +13,7 @@ class RouteHandler(APIHandler):
     # Jupyter server
     @tornado.web.authenticated
     def get(self, path):
-        if path == "available-advanced-links":
+        if path == "advanced-links":
```

Comment:
I would update the `route_pattern` to include the path instead of checking for the path inside the handler

Reply:
Do you mean removing the generic part of the handler and keeping only the `advanced-links` part, and sending a 404 on other pattern ?

Reply:
yes, your route pattern is something like 

```python
links_route_pattern = url_path_join(base_url, "glue-lab", "advanced-links")
```
You need to add other route patterns and handlers if you want to support other endpoints.

Reply:
Done in https://github.com/QuantStack/glue-lab/pull/49/commits/5db8a3beacf9c6d8eb06c881bee40f0a3b00f040